tests/browserstack_automation/specs/SignedIn.js

```diff
@@ -0,0 +1,319 @@
+import { driver, expect } from '@wdio/globals';
+import SignedIn from '../page_objects/signedin.page';
+import ReadyPage from '../page_objects/ready.page';
+
+const waitTime = 14000;
+
+describe('Profile(Signed In)', () => {
+
+    // ProfileSignedIn_001
+    it.only('verifySignInButton', async () => {
+      await ReadyPage.load();
+      await driver.pause(waitTime);
+      await SignedIn.signInButton.click();
+      await driver.pause(waitTime);
+      await expect(ReadyPage.getTitleSignUpPopUp).toHaveText('Sign In or Join');
+      await expect(SignedIn.signInWithAppleButton).toBeDisplayed();
+      await expect(SignedIn.signInWithPhoneNumberOption).toBeDisplayed();
+      await expect(SignedIn.signInWithEmailOption).toBeDisplayed();  
+    });
+
+    //ProfileSignedIn_002
+    it.only('verifyOpeningAccountSettings', async () => {
+      await ReadyPage.load();
+      await driver.pause(waitTime);
+      await SignedIn.signInButton.click();
+      await driver.pause(waitTime);
+      await SignedIn.enterPhoneNumberInPhoneNumberInput();
+      await SignedIn.clickOnSendCodeOption();
+      await driver.pause(waitTime);
+      await SignedIn.enterVerificationCode();
+      await SignedIn.clickOnVerifyButtonForCodeVerification();
+      await driver.pause(waitTime);
+      await SignedIn.clickOnProfilePhotoPlaceholder();
+      await driver.pause(waitTime);
+      await expect(driver).toHaveUrl(expect.stringContaining('profile'));
```

Comment:
@Ayushee28 Just a suggestion correct me if I am wrong but there is no need to use driver—all commands and functionality are attached to the browser object by default.So you can use :
await expect(browser).toHaveUrl(expect.stringContaining('profile'));

Reply:
I am not sure about that. I had tried with browser and it gives error sometimes for me. Only driver works fine. Also I saw others have used driver as well in the other specs so added this way

Reply:
@Ayushee28 The use of driver is fine